hikari/api/rest.py

```diff
@@ -1455,9 +1456,9 @@ async def delete_messages(
         channel : hikari.snowflakes.SnowflakeishOr[hikari.channels.TextableChannel]
             The channel to bulk delete the messages in. This may be
             the object or the ID of an existing channel.
-        messages : typing.Union[hikari.snowflakes.SnowflakeishOr[hikari.messages.PartialMessage], hikari.snowflakes.SnowflakeishIterable[hikari.messages.PartialMessage]]
+        messages
```

Comment:
Is the removal intentional?
```suggestion
        messages : typing.Union[snowflakes.SnowflakeishOr[messages_.PartialMessage], typing.Iterable[snowflakes.SnowflakeishOr[messages_.PartialMessage]], typing.AsyncIterable[snowflakes.SnowflakeishOr[messages_.PartialMessage]]],
```

Reply:
Yes. The type would become so big that it would look back both in the file and in the documentation